overlord/devicestate/devicestate.go

```diff
@@ -598,20 +583,16 @@ func Remodel(st *state.State, new *asserts.Model) (*state.Change, error) {
 		return nil, fmt.Errorf("cannot remodel to different series yet")
 	}
 
-	// TODO:UC20: support remodel, also ensure we never remodel to a lower
+	// TODO:UC20: ensure we never remodel to a lower
```

Comment:
isn't this TODO done now so we could just make this a normal comment?

Reply:
The code prevents remodeling to a different grade, but they may be cases related to grade change that we can explore. Although as discussed with @pedronis this is unlikely we would allow changing grade at all.

Reply:
ah okay thanks for clarifying this